docs/projects/SUMMARY.md

```diff
@@ -42,4 +42,5 @@
   * [controller button](/projects/jacdac/controller-button)
   * [LCD screen](/projects/jacdac/lcd-screen)
```

Comment:
do you want to keep all these extensions?

Reply:
I have left for the things we need to support on the board.
Perhaps later removing or adding unnecessary stuff will be handled by our application team.
What I left behind was what they agreed to :)

Reply:
Looking at the ci log, it seems that the testing project is causing unnecessary logs regardless of the configuration on our board.

I will sort out the projects.
I think it's better to proceed with a new PR.

Reply:
I've committed(f4a5fb3) to this PR since the cleanup of the 'projects' folder is done before this PR merges.
If this is complicated, I will reset it and then PR it a new one.